docs/advanced_caching.md

```diff
@@ -95,7 +95,6 @@ Python's [`id`](https://docs.python.org/3/library/functions.html#id) function |
 
 - Speed: Fast
 - Use case: If you're hashing a singleton object, like an open database connection or a TensorFlow session. These are objects that will only be instantiated once, no matter how many times your script reruns.
-- [Example](#example-1-pass-a-database-connection-around)<br/>
```

Comment:
Is this intentional ?

Reply:
Yes. If you go to that page, you will notice that the `example` link is part of the header for all of the example, and this is the only one where it is duplicated as part of the body of the function

![Screenshot from 2020-04-25 14-27-00](https://user-images.githubusercontent.com/2762787/80287856-d070ec80-8701-11ea-9b05-ea53fdf48edc.png)

Reply:
thanks for clarifying